cura/CuraApplication.py

```diff
@@ -161,6 +161,7 @@ class ResourceTypes:
 
     def __init__(self, *args, **kwargs):
         super().__init__(name = "cura",
+                         app_display_name = "Ultimaker Cura",
```

Comment:
Could this be retreived from CuraVersion.py(.in)?

Reply:
Good point. I'll change that.